### PR TITLE
[1799] Make jwt public shared by different services

### PR DESCRIPTION
**Pull request by PhucDuongBitmark**
**Description**

- Story link: https://github.com/bitmark-inc/autonomy/issues/1799
- Extra information about this PR:

**Checks**

- [ ] Push jwt public key to aws parameter store
- [ ] redesign a way to fetch and key for validate JWT

### CODE REVIEW

**Comment by PhucDuongBitmark** [comment]
@lemonlatte @hvthhien please help me check this pr.

File: parameter.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	"context"
+	"crypto/rsa"
+
+	"github.com/dgrijalva/jwt-go"
+
+	"github.com/bitmark-inc/config-loader/external/aws/ssm"
+)
+
+// GetRSAPublishKeyFromParameterStore get RSA Publish Key from Parameter Store
+func GetRSAPublishKeyFromParameterStore(ctx context.Context, parameterName string, parameterStore *ssm.ParameterStore) (*rsa.PublicKey, error) {
```
Comment (hvthhien) [inline parameter.go]:
```suggestion
func GetRSAPublicKeyFromParameterStore(ctx context.Context, parameterName string, parameterStore *ssm.ParameterStore) (*rsa.PublicKey, error) {
```

Comment (PhucDuongBitmark) [reply parameter.go]: fixed, please check that

**Review by lemonlatte — APPROVED** [review]
LGTM

**Pull request merged**